cpp/include/cugraph/utilities/dataframe_buffer.hpp

```diff
@@ -202,6 +202,15 @@ auto get_dataframe_buffer_begin(BufferType& buffer)
     std::make_index_sequence<std::tuple_size<BufferType>::value>(), buffer);
 }
 
+template <typename BufferType,
+          typename std::enable_if_t<is_std_tuple_of_arithmetic_device_spans<
+            std::remove_cv_t<BufferType>>::value>* = nullptr>
+auto get_dataframe_buffer_begin(BufferType& buffer)
+{
+  return detail::get_dataframe_buffer_begin_tuple_impl(
+    std::make_index_sequence<std::tuple_size<BufferType>::value>(), buffer);
+}
```

Comment:
If we plan to add this to support std::tuple of device_span objects, we need to add for cbegin, end, and cend as well.